core-kotlin-modules/core-kotlin-collections-map/src/test/kotlin/com/baeldung/removekey/RemoveKeyDuringIterationUnitTest.kt

```diff
@@ -0,0 +1,63 @@
+package com.baeldung.removekey
+
+import org.junit.jupiter.api.Test
+import java.util.ConcurrentModificationException
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+
+class RemoveKeyDuringIterationUnitTest {
+
+    @Test
+    fun `test removing key from map while iterating using for loop should pass`() {
```

Comment:
Instead of saying "should pass" let's specify what the expectation actually is, something like:
```suggestion
    fun `test removing key from map while iterating using for loop should remove successfully`() {
```

Similar for other tests.

Reply:
Thank you for suggesting this change. I've incorporated these and revised the PR.